streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -355,4 +372,9 @@ private long computeRecordSize(final Bytes key, final ContextualRecord value) {
         }
         return size;
     }
+
+    private void updateBufferMetrics() {
```

Comment:
Should we call this in `init()` and/or `close()`, too?

Reply:
sure; I don't think they're strictly necessary, but it doesn't hurt, and it'll make the system more resilient to future changes.